extension-workspace-references.md

```diff
@@ -43,9 +102,9 @@ _Request_
  */
 interface WorkspaceReferencesParams {
     /**
-     * Metadata about the symbol that is being searched for.
+     * Known properties about the identity of the symbol (for example, the package identifier)
      */
-    query: Partial<SymbolDescriptor>;
+    query: Partial<SymbolIdentifier>;
```

Comment:
`WorkspaceReferenceParams` previously had a field `hints` that mirrored the `hints` field in `DependencyReference`. This scoped the reference search by those that matched some property of the dependency. Its motivation was performance—in many cases, a repository-wide reference search is quite expensive, but if we take advantage of information from the `DependencyReference` (e.g., which files or directories depend on the dependency), we can significantly reduce the amount of work required.